src/worker.c

```diff
@@ -357,12 +362,28 @@ void pg_net_worker(__attribute__ ((unused)) Datum main_arg) {
   proc_exit(EXIT_FAILURE);
 }
 
+#if PG15_GTE
+static Size net_memsize(void) {
+  return MAXALIGN(sizeof(WorkerState));
+}
+
+static void net_shmem_request(void) {
+  if (prev_shmem_request_hook)
+    prev_shmem_request_hook();
+
+  RequestAddinShmemSpace(net_memsize());
+  RequestNamedLWLockTranche("pg_net", 1);
```

Comment:
I think `pg_net` doesn't need to call `RequestNamedLWLockTranche()` since it doesn't have its own LWLocks and for example doesn't call `GetNamedLWLockTranche()` during startup.

Reply:
```suggestion
```